src/main/java/com/desiato/puresynth/services/CustomAuthenticationService.java

```diff
@@ -0,0 +1,35 @@
+package com.desiato.puresynth.services;
+
+import com.desiato.puresynth.models.User;
+import com.desiato.puresynth.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Service
+public class CustomAuthenticationService {
```

Comment:
This is a bad name for a class. Why do you want to add `Custom` in it? In some way, every service or class is `custom`.